### PR TITLE
Pte 13 1 lobby synchronization

### PR DESCRIPTION
**Pull request by Hidennnn**
Players can connect with server and join lobby. Everything is safe

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+NEW_GAME = 1
+
+SERVER_IP = "127.0.0.1"
+SERVER_PORT = 6001
+BUFFER_SIZE = 4096
+
+NUMBER_OF_THREADS_TO_CANCEL_CONNECTION = 1
+NUMBER_OF_STARTED_CONNECTIONS_INIT_VALUE = 0
+NUMBER_OF_STARTED_GAMES_INIT_VALUE = 0
+NOT_NEED_OF_NEW_CONNECTIONS = 0
```
Comment (Misiek22113) [inline server.py]: I would like you to change that, we don't really use verbs like need, actual etc. in constants.

Comment (Misiek22113) [inline server.py]: You Could group this constants like
--Server consts--
.
.
.
--Thread consts--
.
.
.
--Connections-- 
etc.

**Review by Misiek22113 — DISMISSED** [review]

File: server.py (hunk 1 above)
Comment (Hidennnn) [reply server.py]: Done

Comment (Hidennnn) [reply server.py]: Yeah, you right. Grouped and commented

**Review by IgorJonski — APPROVED** [review]

**Pull request merged**